hbase/Dockerfile

```diff
@@ -13,6 +15,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # zip required for the log4shell stuff
 RUN microdnf update && \
     microdnf install \
+    gettext \
```

Comment:
I thought I commented on this before but must have forgotten to submit.
Can you add a comment what this is needed for please?

Reply:
commented in 39a6a9e20136ec89cb169d7cff5dedc050ba50a0

Reply:
Thank you! Even better than what I did